stubs/beautifulsoup4/bs4/element.pyi

```diff
@@ -247,7 +247,7 @@ class Tag(PageElement):
         name: str | None = ...,
         namespace: str | None = ...,
         prefix: str | None = ...,
-        attrs: dict[str, str] | None = ...,
+        attrs: dict[str, Any] | None = ...,
```

Comment:
```suggestion
        attrs: dict[str, str] | None = ...,
```

Reply:
is there action from my side on this?

Reply:
No, I've already applied the suggestion here :)